src/routes/RemediationDetails.js

```diff
@@ -94,7 +94,17 @@ const RemediationDetails = ({
 
   const getDisabledStateText = () => {
     if (!context.permissions.execute) {
-      return 'You do not have the required execute permissions to perform this action.';
+      if (isOrgAdmin()) {
```

Comment:
You could also do something like this:
`Executing the playbook requires having the remediations:remediation:execute permission which is included in the Remediations administrator role. ${isOrgAdmin() ? 'Manage your roles in User access.' : 'Contact your Organization Administrator for access.'}`
Which would remove the string duplication & the if / else statement
What do you think ? @leSamo

Reply:
Sorry for the late response 😅